src/resources/public/css/style.css

```diff
@@ -103,6 +96,34 @@ body {
   color: var(--color-gray-lighter);
 }
 
+.app-input.slug,
+.app-input.password {
+  color: var(--color-yellow);
+}
+
+.app-input-container {
+  display: flex;
+}
+
+.eye-password {
+  position: absolute;
+  cursor: pointer;
+  font-size: 1.2rem;
+  right: 2.5rem;
+}
+
+.eye-signin {
+  top: 12.3rem;
+}
+
+.eye-signup-1 {
+  top: 12.3rem;
+}
+
+.eye-signup-2 {
+  top: 16.9rem;
+}
+
```

Comment:
Do not use number to mark element, specify element functionality in class name like:
```
.signup-button-icon
.signup-button-icon-left
...
```